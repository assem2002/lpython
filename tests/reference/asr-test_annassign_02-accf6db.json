{
    "basename": "asr-test_annassign_02-accf6db",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_annassign_02.py",
    "infile_hash": "419ee1ffc46f3b9fd99f008d4fdde20f4cc65cf82bbd234ad9fcc005",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_annassign_02-accf6db.stderr",
    "stderr_hash": "ec335ab50f2f7d0392f13af981bfa42f115e4d9408ab8f06d178e000",
    "returncode": 2
}