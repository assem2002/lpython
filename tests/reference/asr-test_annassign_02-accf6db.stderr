semantic error: The type 'Pattern' is undeclared.
 --> tests/errors/test_annassign_02.py:2:15
  |
2 |     hex_pat : Pattern[str] = r'-?0[xX]+'
  |               ^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
